keyboards/mechboards/lily58/pro/keymaps/default/rules.mk

```diff
@@ -0,0 +1,2 @@
+ENCODER_MAP_ENABLE = yes
+CONVERT_TO = rp2040_ce
```

Comment:
```suggestion
```

Reply:
With rp2040 controllers being almost the default choice and is there an issue with allowing this?

Reply:
As far as I'm aware, converters at the (`default`) keymap level aren't allowed -- creating an `rp2040_ce` revision or the like would be the best approach.

This would also probably cause confusion for QMK Configurator users who may be expecting an RP2040 UF2 file to be made when compiling the default keymap, but end up with ATmega32U4 hex file.

Reply:
I didn't think configurator supported keyboard level `CONVERT_TO` though?

Reply:
Just linking this for any future readers.
https://discord.com/channels/440868230475677696/867530715192885269/1342616058854637578

It's a firm no to converters in the default keymap.